shop/src/Routes/index.jsx

```diff
@@ -0,0 +1,35 @@
+import { BrowserRouter as Router, Routes, Route } from "react-router-dom";
+import HomePage from "../Pages/HomePage";
```

Comment:
use [React.lazy](https://react.dev/reference/react/lazy). instead for these pages import

Reply:
Yep, but I have not understand about react.lazy 100% yet.